spm_BIDS.m

```diff
@@ -207,7 +207,7 @@
     if isempty(f), f = {}; else f = cellstr(f); end
     for i=1:numel(f)
         
-        p = parse_filename(f{i}, {'sub','ses','task','acq','rec','fa','echo','inv','run','recording', 'meta'});
+        p = parse_filename(f{i}, {'sub','ses','task','acq','ce','rec','dir','run','echo','recording','part','chunk','meta'});
```

Comment:
no part, chunk or echo for physio (also note that this assumes that this physio is for func data)

I have set up the starter kit to give you all the BIDS filename templates:

https://bids-standard.github.io/bids-starter-kit/folders_and_files/files.html#func-functional-mri